test/test.js

```diff
@@ -50,15 +50,15 @@ describe('app.js', function () {
   })
 
   // see https://developer.github.com/apps/building-github-apps/authenticating-with-github-apps/#authenticating-as-a-github-app
-  it.only('gets bearer token', async () => {
-    const bearer = await app.appAuth()
+  it('gets bearer token', function () {
+    const bearer = app.appAuth()
     expect(bearer).to.equal(BEARER)
   })
 
   // see https://developer.github.com/v3/apps/#create-a-new-installation-token
   it('gets installation token', async () => {
     nock('https://api.github.com', {
-      reqheaders: {
+      headers: {
         authorization: `bearer ${BEARER}` // installation access token
```

Comment:
I think this must be `reqheaders`: https://github.com/nock/nock#specifying-request-headers